.env.example

```
@@ -9,4 +9,9 @@ SUPABASE_ENDPOINT=https://your-project-id.supabase.co
 SUPABASE_MASTER_API_KEY=your_supabase_service_role_key_here
 
 # Optional: Development environment flag
-NODE_ENV=development
\ No newline at end of file
+NODE_ENV=development
+
+# Feature Flags
+# Enable/disable the Infobase MDX-based wiki feature
+# Default: enabled in development, disabled in production
+VITE_FEATURE_INFOBASE=true
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)